src/components/templates/movieList/MovieList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { FullMovieList } from "../../organisms";
+import { Content } from "antd/lib/layout/layout";
+import { Typography } from "antd";
+
+function MovieList({
```

Comment:
no need of this template, we can add the heading to the movieList organism

Reply:
MovieList is removed from app